Scripts/Connector/Sample connector backup script.ps1

```diff
@@ -0,0 +1,42 @@
+# This script creates a backup for the connector in the machine in which it is run.
+# The files generated by this script will be saved in C:\ConnectorBackup, these files will
+# need to be copy to the machine in which you wish to restore the Connector.
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+
+# To restore a Connector look at "Sample connector restore script.ps1"
```

Comment:
consider having a warning like this that requires them to press a key to acknowledge:
"this is not intended to create a redundant connector to run in parallel with this connector. Attempting to do so will cause printing failures. It is only intended to back up a connector so it can be restored in case of machine failure or moving to another machine"